include/swift/Serialization/ModuleFormat.h

```diff
@@ -1314,6 +1314,8 @@ namespace decls_block {
   using SynthesizedProtocolDeclAttrLayout
     = BCRecordLayout<SynthesizedProtocol_DECL_ATTR>;
   using ImplementsDeclAttrLayout = BCRecordLayout<Implements_DECL_ATTR>;
+  using NSKeyedArchiveLegacyDeclAttrLayout
+    = BCRecordLayout<NSKeyedArchiveLegacy_DECL_ATTR>;
```

Comment:
We need the stub there because of various bits of preprocessor metaprogramming; note that the attribute above it (`ImplementsAttr`, `SynthesizedProtocolAttr`) are also not serialized.